mart/attack/adversary.py

```diff
@@ -162,38 +181,59 @@ def forward(
         self,
         input: Union[torch.Tensor, tuple],
         target: Union[torch.Tensor, Dict[str, Any], tuple],
-        model: torch.nn.Module,
-        **kwargs
+        *,
+        model: Optional[torch.nn.Module],
```

Comment:
Is `model` actually optional? If so, it needs a default. If not, then remove the `Optional` type.

Reply:
Removed.